### ISSUE TITLE
Mixer overloaded error after upgrading to 1.1.0

### ISSUE THREAD

**Issue by Stono** [istio/istio#12659]
`1.0.6` proxies following a control plane upgrade to `1.1.0` are currently logging:

```
[libprotobuf ERROR src/istio/mixerclient/report_batch.cc:83] Mixer Report failed with: UNAVAILABLE:Server is currently overloaded. Please try again.
[libprotobuf ERROR src/istio/mixerclient/report_batch.cc:83] Mixer Report failed with: UNAVAILABLE:Server is currently overloaded. Please try again.
[libprotobuf ERROR src/istio/mixerclient/report_batch.cc:83] Mixer Report failed with: UNAVAILABLE:Server is currently overloaded. Please try again.
[libprotobuf ERROR src/istio/mixerclient/report_batch.cc:83] Mixer Report failed with: UNAVAILABLE:Server is currently overloaded. Please try again.
```

Every second.

These errors are no present in `1.1.0` sidecars.

**Comment by douglas-reid** [comment]
@Stono you'll need to tune your loadshedding settings, it appears. That's more of an art than it is a science. I believe the default configuration is to reject gRPC requests if P50 latency > `100ms`.  If you are constantly stuck there, I'd bump that up. You could try something like `500ms` as a shot in the dark, but it'd be better to turn the loadshedding into `logonly` mode, watch the latency on `Report` calls for awhile, and then set a limit that matches your environment, watch the logs again, and then turn on `enforce` when you believe you have a good set of limits.

/cc @mandarjog

**Comment by Stono** [comment]
Well we don't see it in 1.1.0 sidecars so I don't think I need to tune anything as we're upgrading them all anyway

**Comment by douglas-reid** [comment]
This should impact both 1.0.X and 1.1 in the same manner, as they should both be hitting the same `istio-telemetry` workloads, correct?  Load-shedding, in this case, is handled server-side.

**Comment by Stono** [comment]
They are, but we don't see this in 1.1.0 sidecars....

**Comment by Stono** [comment]
Closing this as it feels like it won't get fixed, and I didn't see it going from 1.1.6 to 1.1.7, and 1.x is EOL
